src/Essentials/src/Screenshot/Screenshot.uwp.cs

```diff
@@ -32,32 +28,59 @@ public Task<IScreenshotResult> CaptureAsync(Window window) =>
 
 		public async Task<IScreenshotResult> CaptureAsync(UIElement element)
 		{
-			var bmp = new RenderTargetBitmap();
+			if (element is WebView2 webView)
+			{
+				InMemoryRandomAccessStream stream = new();
+				await webView.CoreWebView2.CapturePreviewAsync(Web.WebView2.Core.CoreWebView2CapturePreviewImageFormat.Png, stream);
+
+				BitmapDecoder decoder = await BitmapDecoder.CreateAsync(stream);
+				PixelDataProvider provider = await decoder.GetPixelDataAsync();
+				byte[] byteArray = provider.DetachPixelData();
 
-			// NOTE: Return to the main thread so we can access view properties such as
-			//       width and height. Do not ConfigureAwait!
-			await bmp.RenderAsync(element);
+				return new ScreenshotResult((int)decoder.PixelWidth, (int)decoder.PixelHeight, byteArray, decoder.DpiX, decoder.DpiY);
+			}
+			else
+			{
+				var bmp = new RenderTargetBitmap();
 
-			// get the view information first
-			var width = bmp.PixelWidth;
-			var height = bmp.PixelHeight;
+				// NOTE: Return to the main thread so we can access view properties such as
+				//       width and height. Do not ConfigureAwait!
+				await bmp.RenderAsync(element);
 
-			// then potentially move to a different thread
-			var pixels = await bmp.GetPixelsAsync().AsTask().ConfigureAwait(false);
+				// get the view information first
+				var width = bmp.PixelWidth;
+				var height = bmp.PixelHeight;
 
-			return new ScreenshotResult(width, height, pixels);
+				// then potentially move to a different thread
+				IBuffer pixels = await bmp.GetPixelsAsync().AsTask().ConfigureAwait(false);
+
+				return new ScreenshotResult(width, height, pixels);
+			}
 		}
 	}
 
 	partial class ScreenshotResult
 	{
-		readonly byte[] bytes;
+		readonly double _dpiX;
+		readonly double _dpiY;
+		readonly byte[] _bytes;
+
+		public ScreenshotResult(int width, int height, byte[] bytes, double dpiX, double dpiY)
```

Comment:
Since this is a new API, we will have to keep this internal for main - but make it public once we merge up to net9.0.